components/core/src/ffi/ir_stream/encoding_methods.cpp

```diff
@@ -267,6 +267,20 @@ namespace ffi::ir_stream {
 
         bool encode_message (epoch_time_ms_t timestamp_delta, string_view message, string& logtype,
                              vector<int8_t>& ir_buf)
+        {
+            if (!encode_message(message, logtype, ir_buf)) {
+                return false;
+            }
+
+            if (!encode_timestamp(timestamp_delta, ir_buf)) {
+                return false;
+            }
+
+            return true;
+        }
+
+        bool encode_message (std::string_view message, std::string& logtype,
```

Comment:
We don't need `std::` on these params since we have `using` declarations at the top of the file.